pkgs/tools/text/u2ps/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, stdenv
+, fetchurl
+, autoconf
+, ghostscript_headless
+}:
+
+stdenv.mkDerivation rec {
+  pname = "u2ps";
+  version = "1.2";
+
+  src = fetchurl {
+    url = "https://github.com/arsv/u2ps/archive/refs/tags/${version}.tar.gz";
+    hash = "sha256-h5eYShnoiaBMrsxflBp0QBAHT5lH/6q0FFZ3y9Eqhpk";
+  };
+
+  buildInputs = [ ghostscript_headless ];
+
+  meta = with lib; {
+    description = "Unicode text to postscript converter.";
+    longDescription = ''
+      U2ps is text to postscript converter similar to a2ps,
```

Comment:
U2ps is **a** text to postscript converter similar to a2ps ...

Reply:
Done.